### ISSUE TITLE
Links in the cart have inconsistent styling (accessibility)

### ISSUE THREAD

**Issue by M-Behr** [intershop/intershop-pwa#1790]
**Actual Behavior**

"Remove item" link is grey, large and underlined.
"Enter a promotion code?" link is black and not underlined.

**Expected Behavior**

Links with icons have the primary color and are not underlined. They are underlined by hover.
See "Quick Order" link as example.

**Steps to Reproduce the Bug**

1. Add a quote items to the cart

**Additional Context, like Screenshots, Log File Snippets etc.**

![Image](https://github.com/user-attachments/assets/7bee985a-2fb9-4a71-a629-8397e6065ebd)

[AB#105784](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/105784)
